### PR TITLE
Bootstrap project

### PR DESCRIPTION
**Pull request by Emmanuelaaron**
**Using Bootstrap - Solo project:** 

In this project, I built a clone of The [Newsweek](https://www.newsweek.com/) home page using bootstrap 

Following are some specifications of our project: 
- breaking points at (mobile view)320px,  (tablet view)768px and (Large screen)1440px;

### CODE REVIEW

**Comment by majovanilla** [comment]
# Code Review in progress 👀 by Maria Reyes

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
-# News-week-clone
\ No newline at end of file
+#News week clone
+
+> This is  a clone of [Newsweek](https://www.newsweek.com/).
+> It uses Twitter's bootstrap framework
+
```
Comment (majovanilla) [inline README.md:5]: - Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.

File: styles/responsive.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    max-width: 140px;
+    margin-left: 10px;
+  }
+
+  .disappear,
+  .search-input,
+  .trapezium {
+    display: none;
+  }
+}
```
Comment (majovanilla) [inline styles/responsive.css]: Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.

File: styles/responsive.css [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  }
+
+  .min-content-height {
+    height: max-content;
+  }
+
+  .sign-up-section {
+    margin: 0 auto;
+  }
+
```
Comment (majovanilla) [inline styles/responsive.css]: Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.

File: styles/responsive.css [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+@media screen and (max-width: 320px) {
+  .btn-primary {
+    max-width: 72px;
+    font: bold 11px/7px "Roboto Condensed", sans-serif;
+  }
+
+  .bg-red {
+    justify-content: space-between;
+  }
```
Comment (majovanilla) [inline styles/responsive.css]: Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.

File: styles/responsive.css [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    font-size: 15px;
+  }
+
+  #mobile-btn {
+    max-width: 116px;
+    height: 38px;
+  }
+
+  .newsletter-img {
+    width: 100%;
```
Comment (majovanilla) [inline styles/responsive.css]: Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.

File: styles/style.css [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+}
+
+.list-bk-color {
+  background-color: #fff;
+  padding: 0;
+}
+
+.nav-item {
+  margin: 0 auto;
+}
```
Comment (majovanilla) [inline styles/style.css:35]: Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.

File: styles/style.css [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+
+.list-bk-color li {
+  list-style: none;
+}
```
Comment (majovanilla) [inline styles/style.css:39]: Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.

File: styles/style.css [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+
+.search-input {
+  background-image: url('https://g.newsweek.com/www/images/icon-search-glass.svg');
+  background-repeat: no-repeat;
+  background-position: 100% 50%;
+  background-size: 24px 24px;
+  height: 32px;
+  border: none;
+  width: 90px;
+}
```
Comment (majovanilla) [inline styles/style.css]: Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.

File: styles/style.css [hunk 9]
```diff
@@ -0,0 +1,270 @@
...
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: bold;
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.list-item {
+  height: 134px;
+  padding: 20px 0;
+}
```
Comment (majovanilla) [inline styles/style.css:89]: Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.

File: styles/style.css [hunk 10]
```diff
@@ -0,0 +1,270 @@
...
+
+h6 {
+  text-transform: uppercase;
+}
+
+.double-line img {
+  width: 75px;
+  height: 75px;
+  margin-top: -20px;
+}
```
Comment (majovanilla) [inline styles/style.css:166]: Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,1117 @@
...
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Vantage</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link">Weather</a>
+        </li>
+      </ul>
+    </div>
+    <input type="text" placeholder="Search" class="col-lg-1 search-input mt-lg-2">
```
Comment (majovanilla) [inline index.html]: Use semantic HTML. In this case, you can change the div that contains your list and change it for a `<nav>` tag

Please check the size and spacing of your `<li>` elements. The original page has a different look (font-size, less padding...)
Yours:
Screenshot from 2020-04-13 08-01-22

Original:
Screenshot from 2020-04-13 08-05-12

File: index.html [hunk 12]
```diff
@@ -0,0 +1,1117 @@
...
+            <p class="p-top-story">The federal government has deployed hundreds of tons of supplies from our national
+              stockpile to
+              locations
+              with the greatest need in order to assist in those areas," the president said Sunday.</p>
+          </div>
+        </section>
+      </div>
+      <div class="col-lg-4 mt-md-5 mt-lg-1 order-3 ">
+        <section id="opinion" class="col-lg-12 min-content-height sticky-top opinion p-0">
+          <h2>opinion</h2>
```
Comment (majovanilla) [inline index.html:308]: Check the looks of this section. Yours has more spacing between elements.
Yours:
Screenshot from 2020-04-13 08-01-27

Original:
Screenshot from 2020-04-13 08-01-33

File: index.html [hunk 13]
```diff
@@ -0,0 +1,1117 @@
...
+                    office
+                    said
+                    in a statement.</p>
+                </div>
+              </div>
+            </section>
+            <section id="culture-travel-mobile" class="row d-md-none mt-5 pt-4">
+              <h2 class="col-12">CULTURE & TRAVEL</h2>
+              <div class="col-md-4 col-lg-4">
+                <img
```
Comment (majovanilla) [inline index.html:525]: Check this section. Your font-size is not the same and the font colors also need some work.
Check the badges on the images, too and add them.
On mobile screens, your  section has no title.

Yours:
Screenshot from 2020-04-13 08-06-42

Original:
Screenshot from 2020-04-13 08-06-49

File: index.html [hunk 14]
```diff
@@ -0,0 +1,1117 @@
...
+            <h3 class="h3-feature-story">Rand Paul Tests Positive for Coronavirus</h3>
+            <p class="p-feature-story">He expects to be back in the Senate after his quarantine period ends," according
+              to the Kentucky
+              senator's
+              Twitter account.</p>
+          </div>
+        </section>
+        <section id="top-stories" class="d-none d-md-block d-lg-block col-md-7 col-lg-8">
+          <h2>top Story</h2>
+          <div>
```
Comment (majovanilla) [inline index.html:130]: The images on this section and the Featured stories should have a white badge with red text on the bottom-left corner. Please add them.

File: index.html [hunk 15]
```diff
@@ -0,0 +1,1117 @@
...
+                the
+                question? Here is a list of museums and other popular tourist attractions that have entire collections
+                or
+                performances available online.</p>
+            </div>
+          </section>
+          <section id="more-stories" class="row">
+            <h2 class="col-lg-12">more stories</h2>
+            <div class="col-lg-12 d-flex list-item mb-md-4">
+              <img
```
Comment (majovanilla) [inline index.html]: Check the spacing on this section. Your articles are more separated than the original page.

On mobile screens, your section has no title and it looks disorganized.
Screenshot from 2020-04-13 08-30-51

Original:
Screenshot from 2020-04-13 08-30-45

File: index.html [hunk 16]
```diff
@@ -0,0 +1,1117 @@
...
+                  collections
+                  or
+                  performances available online.</p>
+              </div>
+            </section>
+
+            <section id="latest-news">
+              <h2>Latest news</h2>
+              <div class="col-lg-12 d-flex list-item latest-news border-top mb-2 mb-md-0">
+                <img
```
Comment (majovanilla) [inline index.html:562]: Check the spacing on this section. Your articles are more separated than the original page.

On medium screens, your images lose the proper sizing.
Screenshot from 2020-04-13 08-29-50

File: index.html [hunk 17]
```diff
@@ -0,0 +1,1117 @@
...
+    </div>
+    <div class="row mt-md-5">
+      <div class="row col-lg-12 magazine justify-content-center mt-lg-5 border-bottom">
+        <div class="icon"></div>
+        <h2 class="mt-2 ml-2">in the magazine</h2>
+      </div>
+      <div class="col-md-3 col-lg-3 mt-5">
+        <img
+          src="https://d.newsweek.com/en/full/1575272/fe-covid-cure-opener.webp?w=466&h=311&l=47&t=40&q=90&f=7d5d8ed57150fd6695e0e7d465be9cf5"
+          alt="" class="w-100">
```
Comment (majovanilla) [inline index.html]: The first image of this section should be taller than the rest of the images. Please fix this,

File: index.html [hunk 18]
```diff
@@ -0,0 +1,1117 @@
...
+          </section>
+        </section>
+      </div>
+
+    </div>
+    <div class="row mt-md-5">
+      <div class="row col-lg-12 magazine justify-content-center mt-lg-5 border-bottom">
+        <div class="icon"></div>
+        <h2 class="mt-2 ml-2">in the magazine</h2>
+      </div>
```
Comment (majovanilla) [inline index.html:619]: The size of the articles in this section is not the same as the original page. Please make them look like a clone. Check everything: sizing, fonts, spacing, etc. 
Yours:
Screenshot from 2020-04-13 08-14-24

Orginal:
Screenshot from 2020-04-13 08-14-30

File: index.html [hunk 19]
```diff
@@ -0,0 +1,1117 @@
...
+        <img
+          src="https://d.newsweek.com/en/full/1547486/lipschutz-living-room-hotel-chelsea.webp?w=397&h=265&f=fb4270bf5fbc2591cc43d24d0377621d"
+          alt="" class="w-100">
+        <h3 class="h3-featured-slides mt-3">Inside the Last Bohemian Apartments of the Storied Hotel Chelsea</h3>
+      </div>
+    </div>
+    <section id="us-sec" class="row border-top mt-5">
+      <h2 class="col-lg-12 h2-bottom">U.S.</h2>
+      <div class="col-md-4 mb-3">
+        <img
```
Comment (majovanilla) [inline index.html:807]: Check the spacing on your categories'  sections. You have a lot of space after each one and before the division border. Try to make them look like the original page.
Yours:
Screenshot from 2020-04-13 08-16-49

Orginal:
Screenshot from 2020-04-13 08-16-39

On medium screens, the elements on the right look cramped (no spacing between images and text) and the font-size is larger on the main articles.
Screenshot from 2020-04-13 08-27-38

On mobile screens, the three articles on the bottom should occupy 100% width.
Screenshot from 2020-04-13 08-27-20

File: index.html [hunk 20]
```diff
@@ -0,0 +1,1117 @@
...
+          </div>
+          <div class="col-md-6 pt-5 d-flex flex-column align-items-center">
+            <h3>The Starting 5</h3>
+            <p id="newsletter-p">See why nearly a quarter of a million subscribers begin their day with the Starting
+              5.</p>
+            <button class="btn btn-primary subscribe p-2 rounded-0 p-0 border" type="submit" id="newsletter-btn">SIGN
+              UP ></button>
+          </div>
+        </div>
+      </section>
```
Comment (majovanilla) [inline index.html]: Your subscribe and newsletter sections need some work: 
- Check the titles, they should be aligned left.
- Check the spacing between them, they should be closer.
- Check the Subscribe background-color: it's different.
- Add the two images to the subscribe section.

Yours:
Screenshot from 2020-04-13 08-18-43

Original:
Screenshot from 2020-04-13 08-18-49

The mobile version is not working correctly. Your Subscribe has a very large image that shouldn't have that size, and the newsletter looks like this:
Screenshot from 2020-04-13 08-25-37

File: index.html [hunk 21]
```diff
@@ -0,0 +1,1117 @@
...
+          <li class="border-right text-center pr-3 ml-1">Editorial Guidelines</li>
+          <li class="border-right text-center pr-3 ml-1">Advertise</li>
+          <li class="border-right text-center pr-3 ml-1">Copyright</li>
+          <li class="border-right text-center pr-3 ml-1">Terms & Conditions</li>
+          <li class="border-right text-center pr-3 ml-1">Privacy Policy</li>
+          <li class="border-right text-center pr-3 ml-1">Cookie Policy</li>
+          <li class="border-right text-center pr-3 ml-1">Terms of Sale</li>
+          <li class="border-right text-center ml-1 pr-3">Archive</li>
+          <li class="border-right text-center pr-3 ml-1">Announcements</li>
+          <li class="ml-1 text-center">Consent preferences</li>
```
Comment (majovanilla) [inline index.html:1120]: Check the font color of these elements and make them look like the original page.
When you are on medium and mobile sizes., your footer elements should be aligned left.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Het @Emmanuelaaron !!

## You did a *great* job on this project! 💜 


- Good Readme file
- Good Pull Request Message

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Your page has a different gutter size (the space between columns) and a different margin size on the edges. That makes your elements stick to the sides of the window when you make it smaller and that shouldn't happen. Please fix this.
Screenshot from 2020-04-13 08-24-05

- Check my review on the commented code below.

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Use semantic HTML in the structure of your HTML file. Favor the use of `<nav>`,  `<article>`,   to enclose the elements on your file. 

- Your project would look sooo much better if you include the hover effects on the links of the page, like the original version.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by uryelah** [comment]
Hello **Isabirye Isabirye**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all the scroll at top sections of the clone on wide screens is looking great. You really nailed down the hardest parts of this project 🌟**

I see that you fulfilled most of the requests from the last TSE but you'll need to revisit some you missed before moving on. Hang in there 💪

---------     

## Required Changes 🏗 👷‍♀ 

- [ ] Make your navbar fixed like in the [original site](https://www.newsweek.com/).

- [ ] The 'Subscribe' section should have two magazine page images like the original one. Compare your version and the original below:
 Screenshot_2020-04-13 Newsweek
Screenshot_2020-04-13 Newsweek(1)

-----

- [ ] Something is leaking from your navbar on wide screens. Make sure to fix the extra red area. 
- [ ] Also add the weather information on the navbar. Remember, you are required to make an extra close clone for this project.
**Your version:**
Screenshot_2020-04-13 Newsweek(2)
**Original version:**
Screenshot_2020-04-13 Newsweek(3)

------

- [ ] Remember to add the topic over the image for the 'Featured Stories' articles and to add some margin under the image or above the title:
**Your version:**
Screenshot_2020-04-13 Newsweek(5)
**Original version:**
Screenshot_2020-04-13 Newsweek(4)
 
-----

- [ ] Pay attention to font family, color, size, and weight. Change your version so it's closer to the original:
**Your version:**
Screenshot_2020-04-13 Newsweek(7)
**Original version:**
Screenshot_2020-04-13 Newsweek(6)

----

**Your version:**
Screenshot_2020-04-13 Newsweek(9)
**Original version:**
Screenshot_2020-04-13 Newsweek(8)


-----

- [ ] Like my colleague observed before, make sure the margin between sections is not too wide. Compare the space between sections in your clone and in the original version:
Screenshot_2020-04-13 Newsweek(11)
Screenshot_2020-04-13 Newsweek(10)


------

- [ ] Make sure the side images from the topics sections don't change width on resize:
Screenshot_2020-04-13 Newsweek(12)
Screenshot_2020-04-13 Newsweek(13)

-----

- [ ] Make sure the bottom sections and footer follow the original page style on medium and small break points:
Screenshot_2020-04-13 Newsweek(15)
Screenshot_2020-04-13 Newsweek(14)


-----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by thsvr** [comment]
### **STATUS: CHANGES REQUESTED** :bulb: :bulb: 

Hi @Emmanuelaaron :smile: :smile: 
### 
**You are doing a great job and you are almost there** :muscle: 

I notice that you implemented some of the requested changes Sarah(TSE) asked you to do :fireworks: but to have this project approved, you need to do all the requested changes. I listed them below, can you please read it carefully, do the changes and then ask for another code review? :smile_cat: 

:bug: As asked by Sarah, you need to _"Make your navbar fixed like in the original site"_. Can you please fix it?

:bug: As asked by Sarah, _"The 'Subscribe' section should have two magazine page images like the original one. Compare your version and the original"_ and please change the size of the magazine on the top. 

:bug: As asked by Sarah, you _"Also add the weather information on the navbar."_ Can you please fix it? 

:bug: As mentioned by Sarah, can you _"Make sure the side images from the topics sections don't change the width on resizing."_ and please fix it?

:bug: As mentioned by Sarah, _"Make sure the bottom sections and footer follow the original page style on medium and small breakpoints:"_ , so I notice they are not similar as the original, for example, on the footer you need to add some margin/padding to it looks close to the original. can you please fix it? 


If you have any questions, don't hesitate to contact me through slack [@thaís Vieira Azevedo](url)

### **Happy coding and good luck on the next review** :computer: 

_Project reviewed by Thaís Vieira (TSE)_

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @Emmanuelaaron  :wave:
-
Good job implementing the changes! :tada: Keep it up.
We, however, do have some few changes to carry out before we close this (see review comment).
##
## [REQUIRED] - FIX - The navbar during breakpoints. 
- During screen breaks at (1037 X 378), the navbar breaks. This is our current status: 👇
(1037 X 378)screen_clone
 This is what we should be having at that breakpoint 👇 
header
- Also, remove the hover effect on the signup links and increase the fonts on the links (Sign in and subscribe) as well. Thanks 

## [REQUIRED] - FIX  - The opinion section
- In our opinion section we have some unwanted spaces between elements: 👇 
opnion_clone
- Let's aim for this compact form👇 
opinion

## [REQUIRED] - FIX - The signup newsletter section
- let's get rid of the unnecessary spacings and increase the colour contrast in this section.
signup_clone
- We are aiming to achieve this: 👇 
signup

## [REQUIRED] - FIX - The small screen header
- One final change we have to perform is the header at the small screen: 👇 
small_screen_header_clone

- Let's shape things up to mat its sister: 👇 
small_screen

That should be all.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Comment by uryelah** [comment]
Hey there, great job with the fixes. I can see you did great progress since the first code review. 
There are still some issues with the responsiveness of the page, especially in small screens.

I highly recommend you to use your browser of choice developer tools for changing the screen width while developing and making sure the layout never breaks. [Check this video for a demonstration on how to use Firefox responsive design tools.](https://youtu.be/qGI27bpCZK4)

---------     

## Required Changes 🏗 👷‍♀ 

- [ ] The 'More stories' section is overlapping its text and images on resizing at small screens. Check it bellow how it looks in your clone and how it looks in the original:
Screenshot_2020-04-15 Newsweek(5)
**Original page:**
Screenshot_2020-04-15 Newsweek(6)


----

- [ ] Something similar is happening at the 'Subscribe' section:
Screenshot_2020-04-15 Newsweek(3)
**Original PageL**
Screenshot_2020-04-15 Newsweek(4)


-----

- [ ] Make The footer icons and text are justified and aligned like the original on small screens:
Screenshot_2020-04-15 Newsweek(2)
**Original page:**
Screenshot_2020-04-15 Newsweek(1)

------

You are doing really great, responsiveness can take a lot of work, but when you are done and have a solid and reliable site on your hands you'll be ready to tackle amazing professional-looking projects.

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @Emmanuelaaron 
### good job so far :clap: and for also making changes from previous reviews. However, you have a few issues to solve before your project could be approved. You are close.
- Your images are not responsive on mobile devices, they need to be proportional
image not proportional
- On mobile devices, the width of the content (image and text to go full width), refer original design
on mobile full width
- On mobile make the image and text align with the rest of the content
out of place on mobile
- On mobile devices, image thumbnail is aligned to the left and text to the right, refer original design
on mobile text are aligned to the right
- Your footer is not designed properly, the background of the subscribe section overlaps both for large, medium and mobile devices
subscribe footer section
- Also, solve issues raised by stickler on CSS

### Put more work on the responsiveness of your application

### make these changes and resubmit for another code review. good luck
- For clarity, contact me on slack @iamraymond
- happy coding!

**Comment by lekegitrepo** [comment]
### @Emmanuelaaron 👏 👏
### Good job for the implementation of the required changes mentioned in the previous review. You are almost there 💪
#### Try to fix 👇
**_1. The subscribe and newsletter: they should be of the same height and you can also reduce the width of the subscribe content a little bit and increase the width of the newsletter content a little bit_**

**_2. The images are missing in the "latest news" section._**

**_kindly make the changes and submit for review._**

**_if you have any doubts or questions contact me on slack._**

### _Reviewed by Temitayo Adeleke_

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
#### Status: Request changes

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved 🏆 🎈 

Hi @Emmanuelaaron, congratulations on fixing all the changes recommended by the previous TSE's 👍 Thanks to the reviews and your hard work, the project has now met all the minimum requirements 😄 

[OPTIONAL] 
- Kindly add your email to the author section of the README file

Go ahead and merge to master, do not forget to submit the project completion form 🙂 
_Happy coding_ 👨‍💻 
Good luck with your next projects 😃 
Daniel Larbi Addo on slack

**Pull request merged**